ecs-agent/manageddaemon/managed_daemon.go

```diff
@@ -134,7 +134,8 @@ func defaultImportAll() ([]*ManagedDaemon, error) {
 	}
 	var thisCommand []string
 	thisCommand = append(thisCommand, "--endpoint=unix://csi-driver/csi-driver.sock")
-	thisCommand = append(thisCommand, "--log_dir=/var/log")
+	thisCommand = append(thisCommand, "--log_file=/var/log/csi.log")
+	thisCommand = append(thisCommand, "--logtostderr=false")
```

Comment:
Will the old logs be cleaned up somehow?

Reply:
klog doesn't handle log rotation.  I'm working on that now (as a separate task)

Reply:
Thanks!

Reply:
update set max file size to 20MB (based on existing ECS agent configuration and to line up with pre-existing implementation.)